docker/k8s/Dockerfile

```diff
@@ -14,9 +14,6 @@ RUN mkdir -p /vt && \
     mkdir -p /vt/web && \
     mkdir -p /vtdataroot/tabletdata
 
-# Copy CA certs for https calls
```

Comment:
Don't we still need this, at least for vttablet?

Reply:
No, I forgot that we're already running in debian/stretch, so that file is already available from the default filesystem

Reply:
I ran some tests and it looks like neither `debian:stretch-slim` nor `debian:stretch` has the certs by default. I think we only had them in `vitess/base` because `vitess/bootstrap` is based on `golang`.

Reply:
Ok, good catch. I was working on a different deploy with a mysql liveness probe when I checked for that file. I forgot it was still using the old k8s container.